src/app.js

```diff
@@ -12,5 +13,10 @@ app.get('/', (req, res) => {
 
 app.use(express.static(path.join(__dirname, '..', 'public')));
 
+app.get('/latest', (req, res) => {
+  request(`https://newsapi.org/v2/top-headlines?country=us&apiKey=${process.env.NEWS_API_KEY}`, (error, response, body) => {
```

Comment:
This will be separated tmw if you want to start modularising ur code from now
or we can leave it for last